conda-store-server/conda_store_server/exception.py

```diff
@@ -2,18 +2,44 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
+import typing
+
 
 class CondaStoreError(Exception):
-    pass
+    """Exception raised by conda store
+
+    Attributes
+    ----------
+    message : str
+        message to output to users
+    """
+
+    def __init__(self, msg: str):
+        self.message = msg
```

Comment:
Is the only place this is used `conda_store_server/_internal/server/views/api.py`? If so is there much advantage in introducing the `message` attribute over the standard `.args`?